packages/p2p-v2/src/components/Wizard/Wizard.tsx

```diff
@@ -0,0 +1,105 @@
+//TODO: Below component to be removed once wizard fom deriv-com/ui is ready
+import React, {
+    Children,
+    cloneElement,
+    isValidElement,
+    MutableRefObject,
+    PropsWithChildren,
+    ReactNode,
+    useCallback,
+    useEffect,
+    useState,
+} from 'react';
+import clsx from 'clsx';
+import Step from './WizardStep';
+import './Wizard.scss';
+
+type TWizard = {
+    className?: string;
+    initialStep: number;
+    nav: ReactNode;
+    onStepChange?: (prop: { [key: string]: number }) => void;
+    selectedStepRef?: () => MutableRefObject<HTMLElement>;
+};
+
+const Wizard = ({
+    children = [],
+    className,
+    initialStep = 0,
+    nav = null,
+    onStepChange,
+    selectedStepRef,
+}: PropsWithChildren<TWizard>) => {
+    const [activeStep, setActiveStep] = useState(0);
+
+    const getSteps = useCallback(() => Children.toArray(children), [children]);
```

Comment:
There's a hook for steps btw, if its applicable we could use this:
https://usehooks-ts.com/react-hook/use-step

Reply:
yes. might be. but as i already mentioned above, this component would be replaced. had already discussed with deriv-com/ui regarding the stepper component.